src/localization/locales/en-US.json

```diff
@@ -202,7 +203,7 @@
         "not_found": "Landscape not found",
         "list_column_website": "Website",
         "list_column_members": "Members",
-        "list_empty": "No Landscapes",
+        "list_empty": "<0><0>First, double check the spelling of the landscape name.</0><1>If your landscape is missing in Terraso, you can <1>add it below</1> and <3>become the landscape manager</3>.</1></0>",
```

Comment:
issue: the number seems off. There are two `<0>`s at the start, two `<1>`s and no `2`.

Reply:
@paulschreiber The tags are 0-indexed and map to each level of the markup, please check the react i18next documentation to understand how this works: https://react.i18next.com/latest/trans-component